docs/README.md

```diff
@@ -17,24 +17,26 @@ You can point your existing Prometheus or [FluentD](https://fluentd.org)/[Promta
 We also plan to support a wide variety of other APIs, such as the Datadog agent.
 
 Creating an Opstrace Cluster requires our [command-line interface](./references/cli.md), which talks directly to your cloud provider.
-It orchestrates the tedious process of setting everything up.
-After your cluster is running, our Controller inside the cluster will maintain things over time.
-All of your data resides safely (and inexpensively) in your own S3 buckets.
+It orchestrates the toilsome process of setting everything up inside your account, for example creating [a variety of resources](https://buildkite.com/organizations/opstrace/pipelines/scheduled-main-builds/builds/1322/jobs/d3de7e2d-95c0-4605-baf9-64ca03e1c9d6/artifacts/747bbf79-8dfe-447c-9d5b-68e04bbcf9d1) (which includes its own VPC to live in).
```

Comment:
suggestion: let's say "cloud account" instead of just "account".

Reply:
the build log is not public

Reply:
Let's add another 'advanced topics' section where we lay out the resources we create. It's a rather constant set of resources by now for AWS. Then let's link to that instead of to a build log :).